module/skill/Walk/tests/Walk.cpp

```diff
@@ -0,0 +1,8 @@
+// Uncomment this line when other test files are added
```

Comment:
this should be removed, and another other test folders. the tools PR removed them